### DISCUSSION TITLE
Properties in ReadOnlyListBase

### DISCUSSION THREAD

**Discussion by ashishsinha24** [MarimerLLC/csla#2354] [Questions]
I have a scenario in which a few properties need to be saved in the parent object which is a ReadOnlyListBase.

But when binding to a ViewModel in Blazor, on refreshsync they reset to their default values.

Is it possible to have properties in a ReadOnlyListBase?

**Comment by rockfordlhotka** [comment]
No, list classes have no support for properties at this time.

**Reply by ashishsinha24** [reply]
I need to have 2-3 properties which wont be persisted, but would be needed at runtime. These properties would be in the parent ReadOnlyListBase. What is a proper way to implement this scenario.

**Reply by danielmartind** [reply]
I usually create a new parent readonly/business base and put the properties there.
